README.md

```diff
@@ -79,11 +79,57 @@ let line = data |> Js.Json.parseExn
 ```
 
 #### Encode and decode Base64
+
+To encode and decode Base64, you can bind to Javascript functions `btoa` and `atob`, respectively:
+
+```ml
+external  btoa : string -> string = "window.btoa" [@@bs.val]
+external  atob : string -> string = "window.atob" [@@bs.val]
+
+let _ =  "Hello World!" |> btoa |> atob |> Js.log
+```
+
 #### Generate random numbers
+
+Use [Random module](http://caml.inria.fr/pub/docs/manual-ocaml/libref/Random.html) to generate random numbers
+
+```ml
+let () = 
+  Js.log @@ Random.int 5
```

Comment:
I'm a bit wary of exposing the `@@` operator to newcomers, unless it's absolutely necessary. Could you replace it with parentheses?